### PR TITLE
Th json

### PR DESCRIPTION
**Pull request by TrentHand**
_Once a pull request is submitted, be sure to assign necessary people for review_

## Status
**READY/IN DEVELOPMENT**
the json is completed with everything submitted through the surveys.  Anything else, individuals will need to do a PR and update the JSON.

All empty spaces have a "#" so they will just reload the page.

## Description
A few sentences describing the overall goals of the pull request's commits.
I've renamed the images and I'm currently still updating the 
## Impacted Files in Application
List general components of the application that this PR will affect:

* list files here
classinfo.json
## Deploy Notes
Notes regarding deployment of the contained body of work.  These should note any
changes and commands needed for a working version.

### CODE REVIEW

File: img/pic/.ignore [hunk 1]
```diff
@@ -0,0 +1 @@
+You have successfully downloaded your photos. You will need to unzip these files onto your computer before using them.
```
Comment (ben-d-marks) [inline img/pic/.ignore]: This file should be deleted

File: classinfo.json [hunk 2]
```diff
@@ -1,202 +1,228 @@
...
+      "portfolioLink": "http://www.alexsimonian.com",
+      "linkedInLink": "#",
+      "aboutMe": "Fascinated with the notion of how self-reference and formal rules allow systems to acquire meaning despite being made of 'meaningless' elements. Oh, I also like coffee.",
+      "professionalPic": "img/pic/alexPro.jpg",
+      "personalityPic": "img/pic/alexFun.jpg"
+    },
+    {
+      "name": "Ali Kimbrell",
+      "githubLink": "https://github.com/alirk",
       "portfolioLink": "http://www.debbiebourne.me",
```
Comment (ben-d-marks) [inline classinfo.json]: Ali might not appreciate this errant link

**Review by ben-d-marks — CHANGES_REQUESTED** [review]
Only two small problems then we're good to go!

File: img/pic/.ignore (hunk 1 above)
Comment (TrentHand) [reply img/pic/.ignore]: I've updated the files.  Good catch!

**Review by ben-d-marks — APPROVED** [review]
Looks good

**Pull request merged**